<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectPropertiesEditor</class>
 <widget class="QWidget" name="ObjectPropertiesEditor">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>1104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="posLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lla">
       <property name="text">
        <string>Ширина-Долгота (глобальные)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Координата на траектории</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="llaCombo">
       <item>
        <property name="text">
         <string>Секунды (0.0001)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Минуты (0.0100)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Градусы (1.0000)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QDoubleSpinBox" name="rotZspin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QComboBox" name="rotCombo">
       <item>
        <property name="text">
         <string>Секунды (0.0001)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Минуты (0.0100)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Градусы (1.0000)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="latSpin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="lonSpin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.990000000000009</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="altSpin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string> м</string>
       </property>
       <property name="minimum">
        <double>-3000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>3000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="4">
      <widget class="QDoubleSpinBox" name="trjCoordspin">
       <property name="suffix">
        <string> м</string>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="ecefXspin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string> м</string>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-40100000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>40100000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ecef">
       <property name="text">
        <string>ECEF</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="rotXspin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="ecefZspin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string> м</string>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-40100000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>40100000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="ecefYspin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string> м</string>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-40100000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>40100000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="2" column="4">
      <widget class="QComboBox" name="ecefCombo">
       <item>
        <property name="text">
         <string>0.01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="rot">
       <property name="text">
        <string>Вращение, град.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="rotYspin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="minimum">
        <double>-90.000000000000000</double>
       </property>
       <property name="maximum">
        <double>90.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="stationBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
